bin/cli.js

```diff
@@ -65,10 +71,14 @@ const argv = yargs
   })
   .options({
     current: {
```

Comment:
Ideally this needs now to become `set-current`. It becomes a major change, but otherwise the naming convention isn't very good.

Reply:
I agree, alternatives could be `--latest-documented-version` to avoid confusion with `current`. Another idea could be to add a `get` command that could accept `get current` and `get reference` which would return respectively the latest documented version, and latest git reference

Reply:
Wouldn't "get reference" be a job for the `git` command, rather than the `versionist` command?

Reply:
In this case the reference you get would be the result of `getChangelogDocumentedVersions`, followed by `getGitReferenceFromVersion`, its not actually walking your git history or anything like that, in fact it could return the name of a reference that does not exist yet.

Reply:
Ahhh, I'd forgotten about https://github.com/resin-io/versionist/blob/master/README.md#getgitreferencefromversion

Reply:
Yeah, then let's have a `get <subcommand>`. :+1:

Reply:
:wave: